--- a/arch/arm64/boot/dts/mediatek/mt7622.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7622.dtsi
@@ -56,9 +56,9 @@
-		opp-1262500000 {
-			opp-hz = /bits/ 64 <1262500000>;
-			opp-microvolt = <1250000>;
-		};
-
-		opp-1350000000 {
-			opp-hz = /bits/ 64 <1350000000>;
-			opp-microvolt = <1310000>;
-		};
+		opp-1350000000 {
+			opp-hz = /bits/ 64 <1350000000>;
+			opp-microvolt = <1310000>;
+		};
+
+		opp-1625000000 {
+			opp-hz = /bits/ 64 <1625000000>;
+			opp-microvolt = <1310000>;
+		};
--- a/drivers/clk/mediatek/clk-mt7622.c
+++ b/drivers/clk/mediatek/clk-mt7622.c
@@ -334,1 +334,1 @@
-	    HAVE_RST_BAR, 21, 0x0214, 24, 0, 0x0214, 0),
+	    HAVE_RST_BAR, 21, 0x0214, 24, 0, 0x028A, 0),
